msctl

```diff
@@ -2483,7 +2504,16 @@ case "$1" in
       exit 1
     fi
   ;;
-  update)
+  update|force-update)
+    # Handle the force option.
+    if [ "$1" = "force-update" ]; then
+      if [ -s $VERSIONS_JSON ]; then
+        printf "Removing cached version manifest.\n"
+      fi
+      rm -f $VERSIONS_JSON
```

Comment:
That sounds like a good idea. The only reason we wouldn't want to rename the manifest is if it has a size of 0 and fails the -s test. In this case, the proper thing to do is delete the file.  Adding a copy command to the _if_ block seems like it would do the job.